src/kit/taosdemo/taosdemo.c

```diff
@@ -2735,15 +2738,15 @@ static int printfInsertMeta() {
                     g_Dbs.db[i].superTbls[j].sampleFile);
             printf("      tagsFile:          \033[33m%s\033[0m\n",
                     g_Dbs.db[i].superTbls[j].tagsFile);
-            printf("      columnCount:       \033[33m%d\033[0m\n",
+            printf("      columnCount:       \033[33m%d\033[0m\n        ",
```

Comment:
extra spaces?

Reply:
when printing the meta info, I want to keep all the line has the same indent